pages/common/git-cola.md

```diff
@@ -0,0 +1,32 @@
+# git cola
+
+> A powerful Git GUI with a slick and intuitive user interface.
+> More information: <https://git-cola.readthedocs.io/en/latest/>.
+
+- Start git cola in amend mode:
+
+`git cola --amend`
+
+- Prompt for a Git repository. Defaults to the current directory:
+
+`git cola --prompt`
+
+- Open the Git repository at mentioned path:
+
+`git cola --repo {{path/to/git-repository}}`
+
+- Apply the path filter to the status widget:
+
+`git cola --status-filter {{filter}}`
+
+- Print the git cola version and exit:
+
+`git cola --version`
+
+- Show usage and optional arguments:
+
+`git cola --help`
+
+- Search for bugs by text content:
+
+`git cola --help-commands`
```

Comment:
Do you have any good reason to include `--version`, `--help` and `--help-commands`? We never do that and I don't get, why we should do it in this case.

Reply:
There weren't many commands available, but i can remove these and push new changes?

Reply:
That's a good idea.

Reply:
I have made the necessary changes, kindly look into it. Thank you.